.gitignore

```diff
@@ -23,6 +23,8 @@ vendor/assets/bower_components/
 /db/schema.rb
 /public/system/*
 /coverage/
+/features/req_cache/*
```

Comment:
it's not ideal as one can end up with lots of files to check in, but the trick is once you have your feature working to check in only the needed changes.   You can do this like so:

```
rm -rf features/req_cache
rm -rf features/vcr_cassettes
git checkout features/req_cache
git checkout features/vcr_cassettes
```

then do a complete test run, and only check in changes in those folders then ...